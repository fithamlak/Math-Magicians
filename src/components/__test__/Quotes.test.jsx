import renderer from 'react-test-renderer';
import Quote from '../../pages/Quotes';

it('Snapshot of the Quote page', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
