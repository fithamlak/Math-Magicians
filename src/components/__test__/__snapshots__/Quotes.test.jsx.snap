// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of the Quote page 1`] = `
<h1
  className="quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding - William Paul Thurston
</h1>
`;
